### PR TITLE
fix: update org update template

### PR DESCRIPTION
**Pull request by littlejiancc**
## What this PR does / why we need it:
fix: update org update template

## Does this PR introduce a user interface change?
❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |      update org update template        |
| 🇨🇳 中文    |        更新组织编辑审计日志      |

## Does this PR need be patched to older version?
✅ Yes(version is required)
/cherry-pick release/1.4

## Which issue(s) this PR fixes:

- [Erda Cloud Issue Link](https://erda.cloud/erda/dop/projects/387/issues/all?id=241234&issueFilter__urlQuery=eyJzdGF0ZUJlbG9uZ3MiOlsiT1BFTiIsIldPUktJTkciLCJXT05URklYIiwiUkVPUEVOIl0sInN0YXRlcyI6WzQ0MDIsNzEwNCw3MTA1LDQ0MDMsNDQwNCw3MTA2LDQ0MDYsNDQwNyw0NDEyLDQ1MzgsNDQxMyw0NDE0LDQ0MTUsNDQxNl0sImFzc2lnbmVlSURzIjpbIjEwMDEyNjEiXX0%3D&issueTable__urlQuery=eyJwYWdlTm8iOjF9&issueViewGroup__urlQuery=eyJ2YWx1ZSI6ImthbmJhbiIsImNoaWxkcmVuVmFsdWUiOnsia2FuYmFuIjoiZGVhZGxpbmUifX0%3D&iterationID=541&type=BUG)
Fixes #

### CODE REVIEW

File: shell/app/modules/org/common/audit-templates.json [hunk 1]
```diff
@@ -902,6 +902,17 @@
...
+    "success": {
+      "zh": "更新了企业[@orgName]",
+      "en": "Company updated [@orgName]"
+    },
+    "fail": {
+      "zh": "",
+      "en": ""
+    }
+  },
+  "updateOrgNew": {
```
Comment (daskyrk) [inline shell/app/modules/org/common/audit-templates.json:915]: Suggest to use new key as `XXv2`

Comment (littlejiancc) [reply shell/app/modules/org/common/audit-templates.json]: done

**Pull request merged**